app.R

```diff
@@ -0,0 +1,843 @@
+                        "United States-Males" = "US-Males"
+                      ),
+                      selected = "China-Females"
+                    ),
+                    radioButtons(
+                      inputId = "cohortRight",
+                      label = "Right side of the cohort pyramid",
+                      choices = c(
+                        "China-Females" = "China-Females",
+                        "China-Males" = "China-Males",
+                        "United Kingdom-Females" = "UK-Females",
+                        "United Kingdom-Males" = "UK-Males",
+                        "United States-Females" = "US-Females",
+                        "United States-Males" = "US-Males"
+                      ),
+                      selected = "China-Males"
+                    )
+                  )
+                ),
+                column(
+                  width = 8,
+                  offset = 0,
+                  plotOutput("pyramidCountrySex") %>%
+                    withSpinner(color = boastPalette[3])
+                )
+              )
+            )
+          )
+        ),
+        #### Time Population Pyramids ----
+        tabItem(
+          tabName = "timePry",
+          ### ISSUE move into a Single Page
+          h2("Time and Population Pyramids"),
+          p("Connecting population pyramids with time allows us to explore changes
+            in a country's population. This allows us to see when there might be
+            large increases in the population as well as when a country might
+            suffer from catesphroic loss of life."),
+          p("Explore both population pyramids for the United Kingdom and the
+            United States. What do you notice? What might explain any big changes?"),
+          fluidRow(
+            column(
+              width = 4,
+              offset = 0,
+              wellPanel(
+                selectInput(
+                  inputId = "actualCountry",
+                  label = "Select a country",
+                  choices = c(
+                    "United Kingdom",
+                    "United States"
+                  ),
+                  selected = "United States"
+                ),
+                sliderInput(
+                  inputId = "yearInterval",
+                  label = "Year",
+                  min = 1900,
+                  max = 2018,
+                  value = 1900,
+                  round = TRUE,
+                  sep = "",
+                  animate = animationOptions(interval = 2000, loop = FALSE)
+                )
+              )
+            ),
+            column(
+              width = 8,
+              offset = 0,
+              plotOutput("timePyramid")
+            )
+          )
+        ),
+        #### Fecundity Rate Page ----
+        tabItem(
+          tabName = "fecundityRate",
+          h2("Fecundity Rate"),
+          p("A common metric for life tables is that a country's fecundity rate.
```

Comment:
"A common metric for life tables is a country's fecundity rate."

Reply:
Fixed